python/scannerpy/database.py

```diff
@@ -244,14 +269,87 @@ def start_cluster(self, master, workers):
             master: ssh-able address of the master node.
             workers: list of ssh-able addresses of the worker nodes.
         """
-        master_cmd = 'python -c "from scannerpy import Database as Db; Db().start_master()"'
-        worker_cmd = 'python -c "from scannerpy import Database as Db; Db().start_worker()"'
-
-        master = self._run_remote_cmd(master, master_cmd)
-        workers = [self._run_remote_cmd(w, worker_cmd) for w in workers]
-        master.wait()
-        for worker in workers:
-            worker.wait()
+
```

Comment:
We could make them static functions on `Database`, but the idea here was that they were meant to be standalone functions that you call for starting a worker/master. The problem with attaching them to the `Database` object is that the `Database` init fn tries to spawn a cluster, which then calls these functions.